def/es6.js

```diff
@@ -108,13 +108,26 @@ def("SpreadElement")
     .field("argument", def("Expression"));
 
 def("ArrayExpression")
-    .field("elements", [or(def("Expression"), def("SpreadElement"), null)]);
+    .field("elements", [or(
+        def("Expression"),
+        def("SpreadElement"),
+        def("RestElement"),
+        null
+    )]);
 
 def("NewExpression")
-    .field("arguments", [or(def("Expression"), def("SpreadElement"))]);
+    .field("arguments", [or(
+        def("Expression"),
+        def("SpreadElement"),
+        def("RestElement")
+    )]);
```

Comment:
Can you give an example of a `NewExpression` that parses with a `RestElement`? My version of Babel/Acorn uses `SpreadElement`.